framework/util/show_coverage.pl

```diff
@@ -87,7 +89,11 @@ =head1 DESCRIPTION
         print "name    covered   lines    coverage", "\n";
     }
 
-    while (<$fh>) {
+    # read all input lines into an array and
+    # sort it to get consistent output
+    @lines = <$fh>;
+
+    foreach (sort(@lines)) {
         chomp;
         @fields = split /,/;
```

Comment:
The indentation on the next two lines is inconsistent.  This is in code you wrote (in commit dbd22176).  Could you please fix that, perhaps in a different pull request (now or later) to keep from polluting this one?